app/controllers/application_controller.rb

```diff
@@ -8,7 +8,7 @@ class ApplicationController < ActionController::Base
   include EffectiveUser
 
   FLASH_KEYS = %w[error info success warning other].freeze
-  FLASH_KEY_MAP = { 'notice' => 'info', 'alert' => 'error' }.freeze
+  FLASH_KEY_MAP = { 'notice' => :info, 'alert' => :error }.freeze
```

Comment:
any time I see types changing with persisted data (ex strings to symbols, persisted in a flash) I worry a bit (because of +1/-1 deploy errors with old boxes and new boxes switching data)

In this case... I believe we are 100% ok because the hash keys that we look up by are staying the same, so the box reading the data will be internally consistent

Reply:
> any time I see types changing with persisted data (ex strings to symbols, persisted in a flash) I worry a bit (because of +1/-1 deploy errors with old boxes and new boxes switching data)
> 
> In this case... I believe we are 100% ok because the hash keys that we look up by are staying the same, so the box reading the data will be internally consistent

I've also been tempted to consider supporting either symbol or string for `type`, which could help address this concern as well.